jstest/resources/etc/tester.py

```diff
@@ -73,7 +73,7 @@ def check_tools(options):
         os.remove(FREYA_LOG)
 
 
-def execute(cwd, cmd, args=[]):
+def execute(cwd, cmd, args=None):
```

Comment:
You should add the following code into the function:

```py
if args is None:
    args = []
```

This is necessary because a command belove:
```py
subprocess.Popen([cmd] + args, stdout=stdout, stderr=stderr, cwd=cwd)
```
merges the args and the command list.

Reply:
Note: the [utils.execute](https://github.com/Samsung/js-remote-test/blob/master/jstest/common/utils.py#L38) function also initializes the `args` variable if its value is `None`.